terraform/applications/pastebin/main.tf

```diff
@@ -0,0 +1,15 @@
+module "pastebin-db" {
+  source = "git::https://github.com/mozilla-it/terraform-modules.git//aws/database"
```

Comment:
Just curious if there a reason you are using `git::https` here but not in other places? Is it the inclusion of `ref=` you are using in other places? Should we use master here too?

Reply:
okay, two questions here:

1. why git::https - just copypasta from the internal module doc. 
2. should we use master? this war is currently in progress in the team, with the don't use master brigade losing the last battle. so its tbd, but the team decision atm is use master.

Reply:
I'm somewhat hostile to master, but this issue isn't the right place for me to discuss that with the team.